source/fundamentals/connection.txt

```diff
@@ -55,8 +55,10 @@ options. In the example, we set two connection options:
 ``maxPoolSize=20`` and ``w=majority``. To learn more about connection
 options, read the :ref:`golang-connection-options` section of this guide.
 
-The following code shows how you can use the sample connection URI in a
-client to connect to MongoDB.
+The following code shows how you can use a connection string in a
+client to connect to MongoDB. In this example, the client uses 
+the :ref:`Stable API <golang-stable-api>` feature, though
+you can connect to MongoDB without this feature.
```

Comment:
Issue:
Each part of the connection string is described in its own paragraph, but the Stable API description is combined in the paragraph that presents the example. This makes it more difficult to find when scanning for it, and loads additional information into the introduction.
 
Suggestion:
This may require adding some additional context around what's needed, but I think separating the part about the Stable API into another paragraph would also improve the existing transition of describing the connection string to showing the construction of a client.

E.g. (don't use this verbatim)
"To connect to MongoDB, instantiate a Client and pass the connection string and any options. You can set the Stable API server option that ensures upgrading to newer MongoDB server versions do not cause any breaking behavior changes. Learn more about the Stable API from ...

The following example shows how you can instantiate a client, pass it a connection string, set the Stable API option, and verify your connection to a MongoDB deployment.
"